src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -20,6 +20,19 @@
 #[AllowDynamicProperties]
 class WP_Theme_JSON_Resolver {
 
+	/**
+	 * Container for keep track of registered blocks.
+	 *
+	 * @since 6.1.0
+	 * @var array
```

Comment:
Add one empty line between `@since` and `@var`

```suggestion
	 * @since 6.1.0
	 *
	 * @var array
```

Reply:
I've checked and the existing variables in this class and in other classes (class-wp-theme.php, class-wp-theme-json.php, etc.) don't use an empty line for this case.

Reply:
WordPress core has an [established DocBlock format](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#docblock-formatting) for inline documentation:
```
/**
 * Summary.
 *
 * Description.
 *
 * @since x.x.x
 *
 * @see Function/method/class relied on
 * @link URL
 *
 * @global type $varname Description.
 * @global type $varname Description.
 *
 * @param type $var Description.
 * @param type $var Optional. Description. Default.
 * @return type Description.
 */
```

Reply:
Got it. It's for the functions. Thanks!